scripts/upload.sh

```diff
@@ -14,9 +14,9 @@ echo "Packaging SAM app"
 if [ "${#PROFILES[@]}" -eq 0 ]
 then
   echo "No resources that require signing found"
-  sam package --s3-bucket="$ARTIFACT_BUCKET" --output-template-file=cf-template.yaml
+  sam package --s3-bucket="$ARTIFACT_BUCKET" --template-file="$TEMPLATE_FILE" --output-template-file=cf-template.yaml
```

Comment:
will sam still look for the defaults if they are not set by the action? I'm guessing that would result in an empty string here

Reply:
Ah, the default is set